### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by be-marc**
* Replaces all tasks with 6x6 point grid task
* Adds missing unit tests

### CODE REVIEW

File: R/ResamplingSpCVBuffer.R [hunk 1]
```diff
@@ -5,6 +5,17 @@
...
+#' @note
+#' The `blockcv::buffering` function was written for species distribution modeling with presence-absence data.
+#' However, the default settings allow to conduct a leave-one-out cross validation
+#' for two-class, multi-class and continuous response data, where each observation is one test set.
+#' For each test, all observations outside the buffer around the test observation are included in the training set.
+#'
+#' The parameter `spDataType = PB` and `addBG` are designed for presence-background data in species distribution modelling.
+#' If `spDataType = PB`, test sets are only created for each presence observation (`task\$postive`).
+#' The option `addBG = TRUE` adds the background data inside the buffer to the corresponding test sets.
+#' For each test set, all observations outside the buffer around the test observation are included in the training set.
```
Comment (pat-s) [inline R/ResamplingSpCVBuffer.R]: Please format to a width of 80. See other PR for instructions.

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+# mlr3spatiotempcv 0.0.0.9005
+
+* Add `spDataType` and `addBG` parameter to ResamplingSpCVBuffer
+* Fix TaskRegrST
+* Fix ResamplingSpCVEnv and ResamplingRepeatedSpCVEnv
+* Add generic unit tests
```
Comment (pat-s) [inline NEWS.md]: I'll do this automatically after merging PRs via [fledge](https://github.com/krlmlr/fledge). This save you updating NEWS.md if you denote all news-worthy items with `*` in the final merge commit.

Comment (be-marc) [reply NEWS.md]: So we can remove this?

File: R/ResamplingSpCVBuffer.R (hunk 1 above)
Comment (be-marc) [reply R/ResamplingSpCVBuffer.R]: I merged the commits from the other PRs.

File: NEWS.md (hunk 2 above)
Comment (pat-s) [reply NEWS.md]: Yes. Usually its better to do version bumps in master and not directly in PRs.

The approach with {fledge} I am doing here only applies to repos for which I am the only one bumping.

Comment (be-marc) [reply NEWS.md]: Done

**Comment by pat-s** [comment]
Can you resolve the conflicts please? :)

**Pull request merged**